ts/Stock/Indicators/LinearRegression/LinearRegression.ts

```diff
@@ -108,7 +108,7 @@ class LinearRegressionIndicator extends SMAIndicator {
                  * @type    {number}
                  * @product highstock
                  */
-                xAxisUnit: void 0
+                xAxisUnit: null
```

Comment:
Please change the doclet to inform users about `null` value above:

```
                 * Default value is the closest distance between two data
                 * points.
```
And change type in doclet: `@type    {number}`


Additionally, add an upgrade note (see 4th point: https://github.com/highcharts/highcharts/tree/master/changelog#writing-content-for-the-changelog )